bucket/ffmpeg-nightly-shared.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "20200227-9b22254",
+    "homepage": "https://ffmpeg.zeranoe.com/builds/",
+    "description": "A complete, cross-platform solution to record, convert and stream audio and video.",
```

Comment:
Nothing explains what shared mean

Reply:
The stable shared builds have the same description
https://github.com/ScoopInstaller/Main/blob/master/bucket/ffmpeg-shared.json